src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.CoTaskMemAlloc.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        [DllImport(Libraries.Ole32)]
+        internal static extern IntPtr CoTaskMemAlloc(UIntPtr cb);
```

Comment:
This seems weird. Why is a P/Invoke into `Ole32` in the `Kernel32` class? Historical I presume?

Reply:
It seems like this is a mistake and the class should be Ole32 since it is brand new usage.

Reply:
It's a mistake.